database/main.go

```diff
@@ -2,89 +2,60 @@ package main
 
 import (
 	"context"
-	"crypto/sha256"
-	"encoding/hex"
-	"errors"
 	"fmt"
-	"os"
-	"slices"
-
 	"obot-platform/database/pkg/cmd"
-
-	"github.com/gptscript-ai/go-gptscript"
-	_ "github.com/ncruces/go-sqlite3/driver"
-	_ "github.com/ncruces/go-sqlite3/embed"
+	"os"
 )
 
-var workspaceID = os.Getenv("DATABASE_WORKSPACE_ID")
-
 func main() {
 	if len(os.Args) != 2 {
 		fmt.Println("Usage: gptscript-go-tool <command>")
 		os.Exit(1)
 	}
 	command := os.Args[1]
+	ctx := context.Background()
 
-	g, err := gptscript.NewGPTScript()
-	if err != nil {
-		fmt.Printf("Error creating GPTScript: %v\n", err)
-		os.Exit(1)
+	workspaceID := os.Getenv("DATABASE_WORKSPACE_ID")
+	if workspaceID == "" {
+		// TODO(njhale): Figure out why DATABASE_WORKSPACE_ID is not set here for the UI tools.
+		workspaceID = os.Getenv("GPTSCRIPT_WORKSPACE_ID")
 	}
-	defer g.Close()
-
-	var (
-		ctx                    = context.Background()
-		dbFileName             = "obot.db"
-		dbWorkspacePath        = "/databases/" + dbFileName
-		revisionID      string = "-1"
-		initialDBData   []byte
-	)
-
-	workspaceDB, err := g.ReadFileWithRevisionInWorkspace(ctx, dbWorkspacePath, gptscript.ReadFileInWorkspaceOptions{
-		WorkspaceID: workspaceID,
-	})
 
-	var notFoundErr *gptscript.NotFoundInWorkspaceError
-	if err != nil && !errors.As(err, &notFoundErr) {
-		fmt.Printf("Error reading DB file: %v\n", err)
-		os.Exit(1)
-	}
+	// Get admin DSN from environment variable
+	adminDSN := os.Getenv("POSTGRES_DSN")
```

Comment:
this will need to get plumbed down as a credential from obot when we decide how postgres for this tool is going to be stood up/configured.